### DISCUSSION TITLE
Please help to remove top image from blog posts

### DISCUSSION THREAD

**Discussion by cryptobandteam** [thuliteio/doks#999] [Q&A] [answered]
Hello, dear community. 

Could you please help with instructions to remove top image from the blog post? I see that Images [] is empty, so I can't understand why do I see a blank image in each blog post.

Blog post

title: "Say eehello to Doks 👋"
description: "Introducing Doks, a Hugo theme helping you build modern documentation websites that are secure, fast, and SEO-ready — by default."
excerpt: "Introducing Doks, a Hugo theme helping you build modern documentation websites that are secure, fast, and SEO-ready — by default."
date: 2020-11-04T09:19:42+01:00
lastmod: 2020-11-04T09:19:42+01:00
draft: false
weight: 50
images: []
categories: ["News"]
tags: ["security", "performance", "SEO"]
contributors: ["Henk Verlinde"]
pinned: false
homepage: false

**Comment by cryptobandteam** [comment, marked as answer]
Please help

**Reply by h-enk** [reply]
Not an option yet, but this will be part of the upcoming Doks release.

For now, comment out or remove in `./layouts/blog/single.html` the following lines (10..15):

```html
  <div class="col-md-13"> 
     <div> 
       {{- .Scratch.Set "fillImage" "1270x715 Center" -}} 
       {{ partial "content/figure.html" . }} 
     </div> 
   </div>
```

**Reply by cryptobandteam** [reply]
Thank you. Removed from the post but still see in the list of posts in blog https://murfalgo2.netlify.app/blog/

**Reply by cryptobandteam** [reply]
Fixed by adding layouts/_default/list.html and removing images by you advise above. Thank you so much for you help
